content/source/docs/cloud/getting-started/policies.html.md

```diff
@@ -54,33 +54,37 @@ vi sentinel.hcl
 The configuration file takes a very basic structure, like this:
 
 ```hcl
-policy "sunny-day" {
+policy "allowed-terraform-version" {
     enforcement_level = "soft-mandatory"
 }
 ```
 
 The above configuration is declaring the following:
 
-* We have a policy named `sunny-day`. Its policy code is located in a file named `sunny-day.sentinel` in the same directory as the `sentinel.hcl` file.
-* The `sunny-day` policy has a `soft-mandatory` [enforcement level](#enforcement-levels).
+* We have a policy named `allowed-terraform-version`. Its policy code is located in a file named `allowed-terraform-version.sentinel` in the same directory as the `sentinel.hcl` file.
+* The `allowed-terraform-version` policy has a `soft-mandatory` [enforcement level](#enforcement-levels).
 
 Multiple `policy` blocks may be defined in the `sentinel.hcl` file to configure more policies.
 
 ### Policy Code
 
-In the example above we created a configuration file which assumes that a `sunny-day.sentinel` policy code file will be present. Sentinel code files must follow the naming convention of `<policy name>.sentinel`. Create a `sunny-day.sentinel` policy file in the same directory as the `sentinel.hcl` file:
+In the example above we created a configuration file which assumes that a `allowed-terraform-version.sentinel` policy code file will be present. Sentinel code files must follow the naming convention of `<policy name>.sentinel`. Create a `allowed-terraform-version.sentinel` policy file in the same directory as the `sentinel.hcl` file:
 
 ```bash
-vi sunny-day.sentinel
+vi allowed-terraform-version.sentinel
 ```
 
-The following policy code can be used to perform the most basic check:
+The following policy code can be used to determine the Terraform version that is in use:
 
-```python
-main = rule { true }
+```sentinel
+import "tfplan"
```

Comment:
Thinking about this a bit. If we specified `tfplan/v2` that would automatically enforce a Terraform version check, correct?

Reply:
@dylanegan the v2 imports do not present themselves if the version of Terraform is less than 0.12.x. so there is no version check.

Reply:
Clarified over Zoom that my thoughts were about importing `tfplan/v2` would itself be a version check if the policy outright fails because the import wouldn't exist.